assets/js/constants.js

```diff
@@ -1,7 +1,8 @@
 export const CLICK_EVENT = "click";
 export const HIDE_CLASS = "hide";
 export const icons = {
-    EDIT: `<i class="fa-solid fa-pen" id="edit"></i>`,
-    DELETE: `<i class="fa-solid fa-trash-can" id="delete"></i>`,
-    DONE: `<i class="fa-solid fa-check" id="done"></i>`,
-  };
\ No newline at end of file
+  EDIT: `<i class="fa-solid fa-pen" id="edit"></i>`,
+  DELETE: `<i class="fa-solid fa-trash-can" id="delete"></i>`,
+  DONE: `<i class="fa-solid fa-check" id="done"></i>`,
+  REVERT: `<i class="fa-solid fa-trash-can" id="revert"></i>`,
```

Comment:
it is always a good practice to represent the variables with clear names.

```suggestion
  EDIT_ICON: `<i class="fa-solid fa-pen" id="edit"></i>`,
  DELETE_ICON: `<i class="fa-solid fa-trash-can" id="delete"></i>`,
  DONE_ICON: `<i class="fa-solid fa-check" id="done"></i>`,
  REVERT_ICON: `<i class="fa-solid fa-trash-can" id="revert"></i>`,
```